src/core/ReactInstanceHandles.js

```diff
@@ -115,8 +115,7 @@ function getNextDescendantID(ancestorID, destinationID) {
   // Skip over the ancestor and the immediate separator. Traverse until we hit
   // another separator or we reach the end of `destinationID`.
   var start = ancestorID.length + SEPARATOR_LENGTH;
-  var i;
-  for (i = start; i < destinationID.length; i++) {
+  for (var i = start; i < destinationID.length; i++) {
```

Comment:
This should be failing lint I thought (for violating theoretical "block level" scoping for `i`)